### PR TITLE
macOS: support Retina displays

### PR DESCRIPTION
**Pull request by gch1p**
I added a custom `Info.plist` template with `NSPrincipalClass` key set to `NSApplication`. That trick makes the app look sharp on Retina displays.

### CODE REVIEW

**Comment by bk138** [comment]
Thank you very much for the contribution! Isn't this already covered by https://github.com/wxWidgets/wxWidgets/pull/743 ? 

Which wxWidgets version are you using for building=

File: src/CMakeLists.txt [hunk 1]
```diff
@@ -41,6 +41,10 @@ add_executable(${executable_name} MACOSX_BUNDLE ${multivnc_SRCS})
 
 target_link_libraries(${executable_name} MultiVNCgui ${wxWidgets_LIBRARIES} wxservdisc vncclient)
 
+if(APPLE)
```
Comment (bk138) [inline src/CMakeLists.txt]: this should go into the above `if (APPLE)` stanza and be surrounded by `if (wxWidgets_VERSION_STRING LESS 3.1)`

Comment (gch1p) [reply src/CMakeLists.txt]: It doesn't work that way:
```
CMake Error at src/CMakeLists.txt:38 (set_target_properties):
  set_target_properties Can not find target to add properties to: MultiVNC
```
I guess it doesn't know about the target yet. Have to be after `add_executable`.

**Comment by gch1p** [comment]
> Seems this is only in wx 3.1 versions, the latest Homebrew version being 3.0.4, [MacPorts has version 3.1.2 though](https://ports.macports.org/port/wxWidgets-3.2/summary).
> 
> Which wxWidgets version are you using for building?

You're right, I'm using the wxwidgets from Homebrew, which is 3.0.4. I'll add a check.

**Comment by bk138** [comment]
Merged with minor modifications to endif() clause and commit msg in 28d1dfda2cd906be24766cd73c0cb4f3dd6920be. Thank you very much! 

May I ask in what context you're using MultiVNC? Which areas need  the most improvement?

**Comment by gch1p** [comment]
> Merged with minor modifications to endif() clause and commit msg in [28d1dfd](https://github.com/bk138/multivnc/commit/28d1dfda2cd906be24766cd73c0cb4f3dd6920be). Thank you very much!

Thanks!

> May I ask in what context you're using MultiVNC? 

I was installing a couple of OSes on a virtual machine (libvirt, kvm) on my remote Linux server and needed to access their display. So I configured virt-install to use vnc for graphics output (`--graphics vnc`), forwarded port to my local host using SSH and... and then I needed to find some good open source VNC client, and your app turned out to be good enough!

> Which areas need the most improvement?

I noticed some problems with key capture. Ctrl+C didn't work in shell. I don't know about other shortcuts. I'd say this needs testing and fixing.

**Pull request closed without merging**